cob_bringup/components/cam3d_r200_rgbd.launch

```diff
@@ -21,4 +21,20 @@
 		<arg name="nodelet_manager" value="$(arg name)_nodelet_manager"/>
 		<arg name="start_manager" value="$(arg start_manager)"/>
 	</include>
+
+
+	<!-- static frames broadcasted by camera driver -->
+	<group if="$(arg sim)">
+		<node pkg="tf2_ros" type="static_transform_publisher" name="stp_rgb_frame" args="0.0 0.0 0.0 0.0 0.0 0.0 1.0 /$(arg name)_link /$(arg name)_rgb_frame"/>
+		<node pkg="tf2_ros" type="static_transform_publisher" name="stp_rgb_optical_frame" args="0.0 0.0 0.0 -0.5 0.5 -0.5 0.5 /$(arg name)_rgb_frame /$(arg name)_rgb_optical_frame"/>
+
+		<node pkg="tf2_ros" type="static_transform_publisher" name="stp_depth_frame" args="-0.000958074233495 0.0589913241565 -5.14313214808e-05 0.0 0.0 0.0 1.0 /$(arg name)_link /$(arg name)_depth_frame"/>
```

Comment:
`-0.000958074233495` ==> `0.0` 
`-5.14313214808e-05` ==> `0.0`
(we don't use nanometers)